### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by maxiride** [algernon-A/Realistic-Population-Revisited#7]
I wanted to dig a bit in customizing the values, however I haven't been able to find reference in the CS wiky.

How do the RICO values interact with each other?

An higher or lower square meter per person how does affect residential inhabitants? The height how does integrate in the computation? And so on and so on :)

I'd propose to realize few paragraphs explaining what each value means and how it is used to compute the RICO settings (if there is a formula or something else)

---
**EDIT**
I now found [this](https://steamcommunity.com/workshop/filedetails/discussion/426163185/483368526570131092/) post from the original author where he explains how the values interact. Isn't exactly as KISS as I'd expected but it's more than nothing.

**Comment by algernon-A** [comment]
Yes, documentation is definitely on the list to do.  The upcoming GUI will also add some transparency to the calculations.
